cmd/blockchaincmd/create.go

```diff
@@ -358,6 +359,19 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 	if err = promptValidatorManagementType(app, sc); err != nil {
 		return err
 	}
+	if createFlags.validatorManagerController == nil {
+		var cancelled bool
+		createFlags.validatorManagerController, cancelled, err = getValidatorContractManagerAddr()
+		if err != nil {
+			return err
+		}
+		if cancelled {
+			return fmt.Errorf("user cancelled operation")
+		}
```

Comment:
we should check that this address control funds, say, that it has some allocations, cause it needs to pay tx fees

Reply:
maybe automatically add funds to it? or probably better, ask the user to give funds or if he desires to add funds